web/src/client/users.js

```diff
@@ -165,16 +152,21 @@ class UsersBaseClient {
    * @return {import ("./dbus").RemoveFn} function to disable the callback
    */
   onUsersChange(handler) {
-    return this.client.onObjectChanged(USERS_PATH, USERS_IFACE, changes => {
-      if (changes.RootPasswordSet) {
+    return this.client.ws.onEvent((event) => {
+      if (event.type === "RootChanged") {
+        const res = {};
```

Comment:
What `res` means?

Reply:
well, I use it as result of data in event. Maybe I should use to something like `changed_data`?

Reply:
or just "changes",